src/assets/scss/components/_section.scss

```diff
@@ -6,99 +6,112 @@
 
 /* Section/Article */
 
-	section, article {
-		&.special {
-			text-align: center;
-		}
-	}
-
-	header {
-		&.major {
-			margin-bottom: (_size(element-margin) * 1.5);
-
-			h2 {
-				font-size: 2em;
-
-				&:after {
-					display: block;
-					content: '';
-					width: 3.25em;
-					height: 2px;
-					margin: (_size(element-margin) * 0.35) 0 (_size(element-margin) * 0.5) 0;
-					border-radius: 2px;
-
-					section.special &, article.special & {
-						margin-left: auto;
-						margin-right: auto;
-					}
-				}
-			}
-
-			p {
-				font-size: 1.25em;
-				letter-spacing: _font(letter-spacing);
-			}
-
-			&.special {
-				text-align: center;
-
-				h2 {
-					&:after {
-						margin-left: auto;
-						margin-right: auto;
-					}
-				}
-			}
-		}
-	}
-
-	footer {
-		&.major {
-			margin-top: (_size(element-margin) * 1.5);
-		}
-	}
-
-	@include breakpoint(small) {
-		header {
-			&.major {
-				margin-bottom: 0;
-
-				h2 {
-					font-size: 1.5em;
-				}
-
-				p {
-					font-size: 1em;
-					letter-spacing: 0;
-
-					br {
-						display: none;
-					}
-				}
-			}
-		}
-
-		footer {
-			&.major {
-				margin-top: 0;
-			}
-		}
-	}
-
-	@mixin color-section($p: null) {
-		header {
-			&.major {
-				h2 {
-					&:after {
-						background-color: _palette($p, border);
-
-						@if $p == 'invert' {
-							@include vendor('background-image', 'linear-gradient(90deg, #{_palette(accent1)}, #{_palette(accent3)}, #{_palette(accent5)})');
-						}
-					}
-				}
-			}
-		}
-	}
-
-	@include color-section;
\ No newline at end of file
+section,
+article {
+  &.special {
+    text-align: center;
+  }
+}
+
+section {
+  &.about {
+    p {
+      text-align: left;
+    }
+  }
+}

```

Comment:
This is the only new addition in this file, the rest is purely formatting changes, auto-applied by prettier but I agree with, think makes it easier to read